### DISCUSSION TITLE
Show buttons on List item mouse over

### DISCUSSION THREAD

**Discussion by aneeskA** [fyne-io/fyne#4963] [General] [answered]
Is there a way to display a button or a list of buttons on mouse over of a list item? Fyne being mobile ready also, I am supposing this is not possible or not considered. Any pointers on this matter would be great.

**Comment by andydotxyz** [comment, marked as answer]
Yes you could do that, as you point out it would only work in desktop where there is a mouse.

You would implement the Hoverable interface and add items to your UI based on those events.

**Reply by aneeskA** [reply]
Thank you! Let me look into this.
